packages/ckeditor5-dev-env/tests/translations/upload.js

```diff
@@ -10,67 +10,82 @@
 const path = require( 'path' );
 const sinon = require( 'sinon' );
 const mockery = require( 'mockery' );
-const transifexService = require( '../../lib/translations/transifex-service' );
-const fs = require( 'fs' );
 
 describe( 'upload', () => {
-	let sandbox;
+	const sandbox = sinon.sandbox.create();
 	let upload;
+	let packageNames;
+	let stubs;
```

Comment:
Yes, they can be in the same line. I will change it.

But I noticed, that if you have few variable declarations in one line, you're not reading them all.